addons/product/static/src/product_catalog/kanban_record.js

```diff
@@ -69,6 +69,7 @@ export class ProductCatalogKanbanRecord extends KanbanRecord {
             product_id: this.env.productId,
             quantity: this.productCatalogData.quantity,
             res_model: this.env.orderResModel,
+            context: this.props.record.context,
```

Comment:
Hey! 

Could you do something [like this](https://github.com/odoo-dev/odoo/commit/f764f24bb77e9c3cc4c595c2ef880fbd28f6f45e#diff-f3d62ec224f34fa2cd13a006ccfd6d2d9a805c0b263600b6679881a0ea169642R71) to avoid giving the context in all modules? 

I think that `projet_purchase` could inject the `project_id` directly from the context in an override in the module.
```suggestion
            project_id: this.props.record.context.project_id,
```